client/src/document/ingredients/browser-compatibility-table/index-desktop-lg.scss

```diff
@@ -0,0 +1,10 @@
+@media screen and (min-width: $screen-lg) {
+  .table-scroll {
+    width: 100%;
+    margin: 0;
+  }
+
+  .table-scroll-inner {
+    padding: 0;
+  }
+}
```

Comment:
Any particular reason to split these files up? Generally I would prefer a mobile first approach so, something like:

```css
/* mobile styles */

....

@media screen and (min-width: $screen-md) {
/* tablet and small desktop */
}

@media screen and (min-width: $screen-lg) {
/* large desktop and up */
}
```

And because we are in a SASS world, I prefer to collocate these with the selector, for example:

```css
/* mobile styles */
.standard-table {
  max-width: 100%;

  @media screen and (min-width: $screen-md) {
    max-width: 80%;
  }

  @media screen and (min-width: $screen-lg) {
    max-width: 100%;
  }
}
```

Reply:
My main intention was to bring some structure into those styles, because I found it pretty hard to navigate in that big file, and found it confusing that some styles that definitely only applied to the mobile version were also applied to desktop and then overwritten (yet not systematically).

Your example works for one specific style, but when we set `display: flex` plus a couple of other flex-styles, and then only overwrite `display: table-cell` in desktop, I'm not really fond of it.

Reply:
> And because we are in a SASS world, I prefer to collocate these with the selector, for example:

And to add to that: After extracting, I noticed that some rules are not scoped to the BCD tables. But maybe there's a trick to keep the overview in these huge style files?

Reply:
As discussed, I opened #5658 to track this.